03_SortedSet/src/cc/jennylam/cs146/SortedSetRepl.java

```diff
@@ -40,6 +41,10 @@ else if (input.charAt(0) == 'a') {
                 System.out.println(list);
             } else if (input.charAt(0) == 'c' && !Character.isAlphabetic(input.charAt(1)))
                 System.out.println(list.contains(Integer.valueOf(input.substring(1))));
+            else if (input.charAt(0) == 'p') {
+              populateList(Integer.valueOf(input.substring(1)));
+              System.out.println(list);
+            }
```

Comment:
Should conform code style to non-block notation for better code style consistency